README.md

```diff
@@ -22,4 +23,13 @@ At the user level, customizing this database involves working with the [schema](
 
 Under the hood, this template leans heavily on [AstrodbKit2](https://github.com/astrodbtoolkit/AstrodbKit2), a codebase built on [SQLAlchemy](https://www.sqlalchemy.org/).
 
+Units
+---------------------------
+Per [Chen et al. 2022](https://iopscience.iop.org/article/10.3847/1538-4365/ac6268), we explicitly define the units
+for each table in their name (e.g., in the Spectra_Jy_per_micron table, the flux units are Jy, and the 
+wavelength units are micron). Doing so removes unit ambiguity when querying and modifying the database.
```

Comment:
Update the example so it's using the `Sources` table rather than using `Spectra`

Reply:
Done!